tests/requirements-libraries.txt

```diff
@@ -54,26 +54,26 @@ requests==2.23.0
 scipy==1.4.1
 # simplejson is used for text_c_extension
 simplejson==3.17.0
-sphinx==2.4.1
+sphinx==2.4.4
 # Required for test_namespace_package
-sqlalchemy==1.3.13
+sqlalchemy==1.3.15
 Unidecode==1.1.1
 zeep==3.4.0
-zope.interface==4.7.1
+zope.interface==5.0.2
 
 
 # Python 3.5 not supported / supported for older versions
 # -------------------------------------------------------
 openpyxl==3.0.3; python_version > '3.5'
-web3==5.6.0; python_version > '3.5'
+web3==5.7.0; python_version > '3.5'
 
 # iPython 7.10.0 and higher dropped Python 3.5 support per https://ipython.readthedocs.io/en/stable/whatsnew/version7.html#stop-support-for-python-3-5-adopt-nep-29.
-ipython==7.12.0; python_version > '3.5'
-ipython==7.9.0; python_version == '3.5'
+ipython==7.13.0; python_version > '3.5'
+ipython==7.13.0; python_version == '3.5'
 
 # pandas 1.0.0 dropped Python 3.5 support.
-pandas==1.0.1; python_version > '3.5'
-pandas==0.25.3; python_version == '3.5'
+pandas==1.0.3; python_version > '3.5'
+pandas==1.0.3; python_version == '3.5'
```

Comment:
Should be `pandas==0.25.3; python_version == '3.5'  # pyup: ignore`.

Reply:
Done in new PR